### PR TITLE
Millisecond Precision Tdl Timestamps

### PR DESCRIPTION
**Pull request by kojiromike**

### CODE REVIEW

File: src/eBayEnterprise/RetailOrderManagement/Payload/Order/TOrderContext.php [hunk 1]
```diff
@@ -25,15 +25,30 @@
...
+     * The fraud system requires millisecond precision.
+     * PHP DateTime does not support this, but we use that for backwards compatibility.
+     *
+     * @param DateTime
+     * @return self
+     */
+    public function setTdlOrderTimestamp(DateTime $tdlOrderTimestamp, $milliseconds = 0)
     {
         $this->tdlOrderTimestamp = $tdlOrderTimestamp;
+        $this->tdlMilliseconds = (int) $milliseconds;
```
Comment (rgabruel) [inline src/eBayEnterprise/RetailOrderManagement/Payload/Order/TOrderContext.php]: Shouldn't this property have its own setter and getter?

**Comment by scottvanbrug** [comment]
As an alternative, I double-checked PHP docs and it looks like date formats do support microseconds with a `u` format character. Precision is to 6 places, so if we have a hard requirement on it being 3 we might still need some amount of hacks to make it work, but could be cleaner than passing separate microseconds when setting the DateTime.

**Comment by kojiromike** [comment]
@scottvanbrug Ah, there's an ancient, highly upvoted comment to the contrary, but you're right. I'll alter the PR.

**Comment by ryaan-anthony** [comment]
that should do it

**Pull request merged**